examples/psf_wf_movie.py

```diff
@@ -0,0 +1,258 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+import warnings
+import numpy as np
+import galsim
+
+try:
+    import matplotlib.pyplot as plt
+    import matplotlib.animation as anim
+except ImportError:
+    raise ImportError("This demo requires matplotlib!")
+try:
+    from astropy.utils.console import ProgressBar
+except ImportError:
+    raise ImportError("This demo requires astropy!")
+
+def simple_moments(img):
+    """Compute unweighted 0th, 1st, and 2nd moments of image.  Return result as a dictionary.
+    """
+    array = img.array
+    scale = img.scale
+    x = y = np.arange(array.shape[0])*scale
+    y, x = np.meshgrid(y, x)
+    I0 = np.sum(array)
+    Ix = np.sum(x*array)/I0
+    Iy = np.sum(y*array)/I0
+    Ixx = np.sum((x-Ix)**2*array)/I0
+    Iyy = np.sum((y-Iy)**2*array)/I0
+    Ixy = np.sum((x-Ix)*(y-Iy)*array)/I0
+    return dict(I0=I0, Ix=Ix, Iy=Iy, Ixx=Ixx, Iyy=Iyy, Ixy=Ixy)
+
+def ellip(mom):
+    """Convert moments dictionary into dictionary with ellipticity (e1, e2) and size (rsqr).
+    """
+    rsqr = mom['Ixx'] + mom['Iyy']
+    return dict(rsqr=rsqr, e1=(mom['Ixx']-mom['Iyy'])/rsqr, e2=2*mom['Ixy']/rsqr)
+
+def make_movie(args):
+    """Actually make the movie of the atmosphere given command line arguments stored in `args`.
+    """
+
+    # Initiate some GalSim random number generators.
+    rng = galsim.BaseDeviate(args.seed)
+    u = galsim.UniformDeviate(rng)
+
+    # The GalSim atmospheric simulation code describes turbulence in the 3D atmosphere as a series
+    # of 2D turbulent screens.  The galsim.Atmosphere() helper function is useful for constructing
+    # this screen list.
+
+    # First, we estimate a weight for each screen, so that the turbulence is dominated by the lower
```

Comment:
Perhaps say why we want the turbulence to be dominated by the lower layers - is it because Jee&Tyson found this led to to a better match with observations, we have some physical reason why this should be right, it's just an arbitrary example, ...?